test/server/cards/02_SHD/units/PrivateerCrew.spec.ts

```diff
@@ -0,0 +1,43 @@
+describe('Privateer Crew', function() {
+    integration(function(contextRef) {
+        describe('Privateer Crew\'s triggered ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['privateer-crew'],
+                    },
+                    player2: {}
+                });
+            });
+
+            it('should not give himself 3 experience while played from hand', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.privateerCrew);
+                expect(context.player2).toBeActivePlayer();
+                expect(context.privateerCrew.isUpgraded()).toBeFalse();
+            });
+        });
+
+        describe('Privateer Crew\'s triggered ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        resources: ['privateer-crew', 'battlefield-marine', 'echo-base-defender', 'tech#source-of-insight', 'daring-raid', 'krayt-dragon', 'atst', 'covert-strength', 'commission', 'hunting-nexu'],
```

Comment:
In the future you can just make every card except the one you need to be 'atst' for easy copy-paste